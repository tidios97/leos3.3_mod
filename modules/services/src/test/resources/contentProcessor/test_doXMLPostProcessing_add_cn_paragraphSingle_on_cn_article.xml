<root xmlns:leos="urn:eu:europa:ec:leos" xmlns:xml="http://www.w3.org/XML/1998/namespace">
    <article xml:id="akn_art_hbaALO" leos:editable="true" leos:deletable="true" leos:softaction="add" leos:softuser="demo"
             leos:softdate="2019-01-29T12:08:32.247+01:00" leos:origin="ec">
        <num xml:id="akn_art_hbaALO_1Onb1M" leos:editable="false">Article -1</num>
        <heading xml:id="akn_art_hbaALO_OMUNZD">Article heading...</heading>
        <paragraph leos:origin="cn" xml:id="akn_art_hbaALO-par1">
            <num xml:id="akn_art_hbaALO_eu7RzT">1.</num>
            <content xml:id="akn_art_hbaALO_p78bjk">
                <p xml:id="akn_art_hbaALO_qKD1jn">Text...</p>
            </content>
        </paragraph>
    </article>
</root>