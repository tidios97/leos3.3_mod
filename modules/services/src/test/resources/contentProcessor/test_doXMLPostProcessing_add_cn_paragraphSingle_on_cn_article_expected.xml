<root xmlns:leos="urn:eu:europa:ec:leos">
    <article leos:deletable="true" leos:editable="true" leos:origin="ec" leos:softaction="add"
             leos:softdate="2019-01-29T12:08:32.247+01:00" leos:softuser="demo" xml:id="akn_art_hbaALO">
        <num leos:editable="false" xml:id="akn_art_hbaALO_1Onb1M">Article -1</num>
        <heading xml:id="akn_art_hbaALO_OMUNZD">Article heading...</heading>
        <paragraph leos:origin="cn" leos:softaction="add" leos:softdate="2022-06-17T09:12:18.023+02:00"
                   leos:softuser="demo(DIGIT)" xml:id="akn_art_hbaALO-par1">
            <num xml:id="akn_art_hbaALO_eu7RzT">1.</num>
            <content xml:id="akn_art_hbaALO_p78bjk">
                <p xml:id="akn_art_hbaALO_qKD1jn">Text...</p>
            </content>
        </paragraph>
    </article>
</root>